deps/tools/common.mk

```diff
@@ -29,6 +31,7 @@ CMAKE_COMMON += -DCMAKE_CXX_COMPILER="$(CXX_BASE)"
 ifneq ($(strip $(CMAKE_CXX_ARG)),)
 CMAKE_COMMON += -DCMAKE_CXX_COMPILER_ARG1="$(CMAKE_CXX_ARG)"
 endif
+CMAKE_COMMON += -DCMAKE_LINKER="$$(which $(LD))"
```

Comment:
some versions of cmake were buggy if you didn't pass an absolute path for CC. not sure about LD, but should test that this doesn't cause any issues with cross compilation